recipes-bsp/grub/grub/grub.d/bootimage.cfg

```diff
@@ -0,0 +1,3 @@
+set otherbootargs="rootwait rw usbcore.usbfs_memory_mb=0 consoleblank=0 rcu_nocbs=all "
+
+set usb_gadget_args="g_ether.idVendor=${USBVendorID} g_ether.idProduct=${USBProductID} g_ether.iProduct=${USBProduct}[${hostname}] g_ether.iSerialNumber=${SerialNum} g_ether.dev_addr=${usbgadgetethaddr} g_ether.bcdDevice=${USBDevice}"
```

Comment:
Just curious, is there any reason not to put this content directly in grub.cfg? All the other variables used in the menu entry are defined locally there. There used to be value in having them in a separate file since they had different values in safemode/runmode, but now the values are always the same.

Reply:
I guess we could put them in grub.cfg directly. I'll address this in another PR.

Reply:
Addressed it in https://github.com/ni/meta-nilrt/pull/128